Automattic/woocommerce-payments
# Transaction Details → Show steps to resolve for inquiries

Fixes #7245 (Inquiry steps to resolve)
Fixed #7255 (Show "Past due" in steps to resolve when dispute past due)

> **Note**
> This PR is behind feature flag `_wcpay_feature_dispute_on_transaction_page`

#### Changes proposed in this Pull Request

Add the 'Steps to resolve' section on the transaction details page for inquiries.

![inquiry-steps](https://github.com/Automattic/woocommerce-payments/assets/3147296/3d5ebdae-b0a7-4769-90b7-282ad9aebf7b)

Notes:
- The step **Review the claim** is not part of this PR's scope – now handled in issue #7156.
- See design: https://www.figma.com/file/RiEQmaKRI7u9USAI3lyZz0VX/Dispute-Flows?type=design&node-id=7137-70997&mode=dev

#### Testing instructions

* Enable the feature flag by running `update_option( '_wcpay_feature_dispute_on_transaction_page', '1' );` in [WP Console](https://wordpress.org/plugins/wp-console/).
* Create a disputed inquiry transaction using the card `4000000000001976` at checkout.
* Open the transaction details page of that new order by going to admin > Payments > Transactions > click on the topmost transaction.
* Confirm there is 'Steps to resolve' section.
* Confirm the steps are:
	```
	1. Email the customer to identify the issue and work towards a resolution where possible.
	2. Submit evidence (?) or issue a refund by (date)
	```
* Click on the 'Email the customer' link. It should open your local email client. Confirm:
  * To address is the customer's email address.
  * Subject is `Problem with your purchase from {store name} on {YYYY-MM-DD of the charge date}?`
  * Body is:
	```
	Hello {customer name}
	
	We noticed that on {YYYY-MM-DD of the dispute created date}, you disputed a {currency formatted dispute amount, eg: $10.00} from {YYYY-MM-DD of the charge date}. We wanted to contact you to make sure everything was all right with your purchase and see if there's anything else we can do to resolve any problems you might have had.
	
	Alternatively, if the dispute was a mistake, you could easily withdraw it by calling the number on the back of your card. Thank you so much - we appreciate your business and look forward to working with you.
	```
* Confirm the 2nd step has the tooltip:
	```
	To submit evidence, provide documentation that supports your case. Keep in mind that submitting evidence doesn't ensure a favorable outcome. If the cardholder agrees to withdraw the inquiry, you'll still need to officially submit your evidence to prevent bank escalation. Learn more (links to `https://woocommerce.com/document/woopayments/fraud-and-disputes/managing-disputes/#inquiries`)
	```
-------------------


**Post merge**
